### PR TITLE
feat(lightspeed): integrate pf-chatbot extension with lighspeed plugin

### PR DESCRIPTION
**Pull request by divyanshiGupta**
Story: https://issues.redhat.com/browse/RHIDP-3197

https://github.com/user-attachments/assets/f432892c-9fe6-422a-8f60-f2633e8ce6a1

### CODE REVIEW

**Review by rohitkrai03 — APPROVED** [review]
/lgtm

**Review by rohitkrai03 — APPROVED** [review]
/approve

File: plugins/lightspeed/src/components/LightspeedChatBox.tsx [hunk 1]
```diff
@@ -2,92 +2,166 @@ import React from 'react';
...
+        timestamp: msg.timestamp,
+      },
+    ])
+    .flat() as MessageProps[];
+
+  const welcomePrompts = [
+    {
+      title: 'Topic 1',
+      message: 'Helpful prompt for Topic 1',
+      onClick: () => handleInputPrompt('Helpful prompt for Topic 1'),
```
[the remark this answers is not in the document]
Comment (divyanshiGupta) [reply plugins/lightspeed/src/components/LightspeedChatBox.tsx:131]: @aprilma419 can you PTAL at this and let us know what should be the welcome prompts be for the the time being?

[the remark this answers is not in the document]
Comment (aprilma419) [reply plugins/lightspeed/src/components/LightspeedChatBox.tsx:131]: Hey @divyanshiGupta, the welcome prompts are like what this [screen](https://www.figma.com/proto/urwU8VqRvHfbxMqxeVknrv/RHDH-Lightspeed---AI-Assistant?page-id=0%3A1&node-id=209-1256&node-type=FRAME&viewport=1635%2C-207%2C0.47&t=nTf5Is0FWtPJNhOg-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=209%3A1256) shows. cc @ShiranHi @benwilcock

**Pull request merged**